lib/seattleflu/id3c/cli/command/etl/redcap_det_childcare.py

```diff
@@ -49,6 +50,17 @@ class StudyArm(Enum):
 SWAB_AND_SEND_SITE = 'ChildcareSwabNSend'
 RADFORD_SITE = 'UWChildrensCenterRadfordCourt'
 SANDPOINT_SITE = 'ChildcareCenter70thAndSandPoint'
+PORTAGE_BAY_SITE = 'UWChildrensCenterPortageBay'
+MINOR_SITE = 'MinorAvenueChildrensHouse'
+MAINTINYTOTS_SITE = 'TinyTotsDevelopmentCenterMain'
+EASTTINYTOTS_SITE = 'TinyTotsDevelopmentCenterEast'
+DLBEACON_SITE = 'DeniseLouieBeaconHill'
+DLMAG_SITE = 'DeniseLouieMercyMagnusonPI'
```

Comment:
In REDCap this one is: Denise Louie- Mercy Magnuson Pl .... that's an 'el' not an 'aye' ... for Magnuson Place
so for our site use:  'DeniseLouieMercyMagnusonPl" instead of 'DeniseLouieMercyMagnusonPI'
Boy, that gets lost in the comment. DeniseLouieMercyMagnusonPl should end in 'el', not 'aye'.

Reply:
Oh yep! Will change I to l (el) 😅